core/gesture.ts

```diff
@@ -368,41 +367,41 @@ export class Gesture {
    * the drag radius is exceeded.  It should be called no more than once per
    * gesture.
    */
-  private updateIsDragging_() {
+  private updateIsDragging() {
     // Sanity check.
-    if (this.calledUpdateIsDragging_) {
+    if (this.calledUpdateIsDragging) {
       throw Error('updateIsDragging_ should only be called once per gesture.');
     }
-    this.calledUpdateIsDragging_ = true;
+    this.calledUpdateIsDragging = true;
 
     // First check if it was a bubble drag.  Bubbles always sit on top of
     // blocks.
-    if (this.updateIsDraggingBubble_()) {
+    if (this.updateIsDraggingBubble()) {
       return;
     }
     // Then check if it was a block drag.
-    if (this.updateIsDraggingBlock_()) {
+    if (this.updateIsDraggingBlock()) {
       return;
     }
     // Then check if it's a workspace drag.
-    this.updateIsDraggingWorkspace_();
+    this.updateIsDraggingWorkspace();
   }
 
   /** Create a block dragger and start dragging the selected block. */
-  private startDraggingBlock_() {
+  private startDraggingBlock() {
     const BlockDraggerClass = registry.getClassFromOptions(
         registry.Type.BLOCK_DRAGGER, this.creatorWorkspace.options, true);
 
-    this.blockDragger_ =
-        new BlockDraggerClass!((this.targetBlock_), (this.startWorkspace_));
-    this.blockDragger_!.startDrag(this.currentDragDeltaXY_, this.healStack_);
-    this.blockDragger_!.drag(this.mostRecentEvent_, this.currentDragDeltaXY_);
+    this.blockDragger =
+        new BlockDraggerClass!((this.targetBlock), (this.startWorkspace_));
+    this.blockDragger!.startDrag(this.currentDragDeltaXY, this.healStack);
+    this.blockDragger!.drag(this.mostRecentEvent, this.currentDragDeltaXY);
   }
 
   // TODO (fenichel): Possibly combine this and startDraggingBlock_.
```

Comment:
Can you remove this or file a bug while you're in the area?

Reply:
Removed, because I don't think it's pointing at useful work: there's no equivalent to `healStack` for bubbles, and (at least for now) we're using the registry for the block dragger.